api/src/main/java/gov/cms/ab2d/api/controller/ErrorHandler.java

```diff
@@ -176,6 +177,9 @@ private static HttpStatus getErrorResponse(Class clazz) {
     public ResponseEntity<JsonNode> handleTooManyRequestsExceptions(final TooManyRequestsException e, HttpServletRequest request) throws IOException {
         HttpHeaders httpHeaders = new HttpHeaders();
         httpHeaders.add(RETRY_AFTER, Integer.toString(retryAfterDelay));
+        if (e.getJobIds() != null) {
```

Comment:
@sharshar-sb @nflearl do we want to update our production database to explicitly limit all contracts to one running job? In sandbox this is very problematic so it isn't something we can add to liquibase.